packages/terra-avatar/src/variants/Avatar.jsx

```diff
@@ -100,14 +104,17 @@ class Avatar extends React.Component {
 
     const attributes = { ...customProps };
     const customStyles = size ? ({ fontSize: size, ...attributes.style }) : attributes.style;
-    const avatarClassNames = cx([
-      'avatar',
-      setColor(alt, color, hashValue),
-      { 'fallback-icon': this.state.fallback },
-      { image: Boolean(image) },
-      { 'is-deceased': isDeceased },
+    const avatarClassNames = classNames(
+      cx([
+        'avatar',
+        setColor(alt, color, hashValue),
+        { 'fallback-icon': this.state.fallback },
```

Comment:
Do we still need `'fallback-icon'` in Avatar..? Since with latest changes default fallback will be initials.
cc : @yuderekyu

Reply:
this class stops the color variant's box-shadow and background-color from appearing with the image. we could re-name this class to `error-fallback` to sync with the latest fallback changes?

<img width="187" alt="Screen Shot 2020-05-14 at 9 37 54 AM" src="https://user-images.githubusercontent.com/14063284/81947823-9ab77900-95c6-11ea-87cd-ddcc434e9890.png">